src/components/Modal.astro

```diff
@@ -52,6 +52,10 @@ const { frontmatter: person, Content } = personMarkdown;
     display: flex;
   }
 
+  html:has(dialog[open]) {
```

Comment:
actually, now I see `:has` is not supported in firefox, hence why it doesn't work there. let me know if there's a difference between the implementation that already exists and this one

Reply:
Oh interesting we went with the same approach. My browser of use is actually Firefox and I introduced this because it wasn't working for me.

Maybe we should merge both 🤔

Reply:
I think we posted at the same time, does this work for you in firefox ? :)

Reply:
Yes, that's the thing I didn't explained well.

I toggled the experimental flag on so that `has` works on Firefox for me, but even with the flag on, the `body` style doesn't fix the issue, I just re-tested this at the current site version. The `html` one does the trick in the preview of this build for me.

I closed it just so you and me, can open a PR with he right rule, which is both joint.